plugins/@grouparoo/salesforce/src/lib/export/model.ts

```diff
@@ -60,10 +74,12 @@ export function getSalesforceModel(
     }
   }
   if (count > 0) {
-    if (refKeys.length !== count) {
-      throw new Error(`All Salesforce reference model data is required`);
+    if (keys.length !== count) {
+      const message =
+        keys[0].indexOf("Reference") > 0
+          ? `All Salesforce reference model data is required`
```

Comment:
And let's switch this one to: `To enable reference data syncing, all related options must be set.`

Reply:
Then maybe we should pass in the word `Group` or `reference` into this function from its caller instead of trying to guess.